test/spock/build.gradle.kts

```diff
@@ -3,6 +3,7 @@ apply(plugin = "groovy")
 dependencies {
     testImplementation("org.codehaus.groovy:groovy-all:2.4.11")
     testImplementation("org.spockframework:spock-core:1.1-groovy-2.4")
+    implementation(project(":test:commonTest"))
```

Comment:
should be first: 
    implementation(project(":test:commonTest"))
    testImplementation("org.codehaus.groovy:groovy-all:2.4.11")
    testImplementation("org.spockframework:spock-core:1.1-groovy-2.4")